matrix-sdk-android/src/main/java/im/vector/matrix/android/api/Matrix.kt

```diff
@@ -62,6 +66,7 @@ class Matrix private constructor(context: Context, matrixConfiguration: MatrixCo
             WorkManager.initialize(context, Configuration.Builder().build())
         }
         ProcessLifecycleOwner.get().lifecycle.addObserver(backgroundDetectionObserver)
+        matrixConfigurationHolder.matrixConfiguration = matrixConfiguration.copy()
```

Comment:
No need for that. You should inject your configuration in the DI tree like we do with the Context. Look at the MatrixComponent

Reply:
Done